docs/examples/ANNOUNCEMENT.md

```diff
@@ -0,0 +1,28 @@
+# It's finally time to get rusty @Students!
+
+We are happy to announce that the Rust Short will take place from `07/10/2024` to `13/10/2024`.
+
+##  🦀 What is Rust?
+
+Did you know that about 70% of severe security bugs are caused by memory corruption? While this has a lot to do with skill-issues, (which we obviously do not have here at 42 hehe), languages that let you play loose with memory are definitely part of the problem. 
+
+Rust is a modern systems programming language focused on safety and performance. It's designed to be a safer alternative to languages like C/C++. Rust's main goal is to prevent memory-related vulnerabilities while maintaining high performance. It introduces a way of thinking about memory management that prevents many common issues.
+### Why Rust is Awesome
+1. **No segfaults:** Rust's ownership system prevents common pointer issues.
+2. **No data races:** Easier, safer concurrency.
+3. **Safety != slow:** Performance on par with C/C++.
+4. **Goodbye Makefiles:** Cargo simplifies project management.
+5. **Growing Demand:** Adopted by major tech companies.
+##  🚀 Recommended Prerequisite
+
+We recommend having completed the `minishell` and `philosophers` circle before participating in the Rust Piscine. These projects provide insights into C concurrency & memory management, and will help you better appreciate Rust's take on solving these challenges. If you haven't done so yet, don't worry—you can still participate! Just be prepared for a steeper learning curve 💪
+
+##  🏊‍♂️ Details
+- **Duration:** 7 days (`07/10/2024` - `13/10/2024`)
```

Comment:
same as above add placeholder not concrete values so that they don't forget to change them on accident